### PR TITLE
Populate user status for one to one conversations via the backend call already

### PR DESCRIPTION
**Pull request by nickvergessen**
### Steps
1. Open Talk
2. Create one-to-one conversations with X users
3. Reload talk front page

### Before
X requests for user statuses (most with 404)

### After
User status is preloaded via conversations API and no additional requests are needed

### CODE REVIEW

File: docs/conversation.md [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
 * Method: `GET`
 * Endpoint: `/room`
+* Data:
+
+    field | type | Description
+    ---|---|---
+    `noStatusUpdate` | int | Whether the "online" user status of the current user should be "kept-alive" or not (default 0)
```
Comment (blizzz) [inline docs/conversation.md:16]: what are the possible values here? Apart from  0, but what does it stand for?

Comment (nickvergessen) [reply docs/conversation.md:16]: 1, but yeah will add it.

**Review by blizzz — APPROVED** [review]
Looks good, one remark

**Review by PVince81 — APPROVED** [review]
:+1: code and approach looks good

**Comment by nickvergessen** [comment]
Fixed the doc issue and did the fixup

**Pull request merged**